### PR TITLE
Slimernalia changes/fixes

### PR DESCRIPTION
**Pull request by EthanStraub**
- Increased festivity from Sigillaria to 1k each (up from 100 each).
- Added more types of wrapping paper, as well as the wrapping paper hat.
- Increased cost of wrapping paper to 1k each (up from 500 each).
- Juvies can now !dance.
- Added in juice boxes.
- Fixed an issue that prevented people from increasing their festivity by gambling SlimeCoin.
- Made slight adjustment to !negaslime, added in !negapool for clarity.

### CODE REVIEW

File: ew.py [hunk 1]
```diff
@@ -665,8 +665,8 @@ def get_vouchers(self):
 
 	def get_festivity(self):
 		data = ewutils.execute_sql_query(
-		"SELECT {festivity} + COALESCE(sigillaria, 0) + FLOOR({coin_gambled} / 1000000000000) FROM users "\
-		"LEFT JOIN (SELECT {id_user}, {id_server}, COUNT(*) * 100 as sigillaria FROM items INNER JOIN items_prop ON items.{id_item} = items_prop.{id_item} "\
+		"SELECT FLOOR({festivity}) + COALESCE(sigillaria, 0) + FLOOR({coin_gambled} / 1000000000000) FROM users "\
```
Comment (magicksRampage) [inline ew.py:669]: festivity is already stored as an int in the database

Comment (EthanStraub) [reply ew.py:669]: I changed it to a float so that it can properly store festivity gains from slimecoin.

Comment (EthanStraub) [reply ew.py:669]: Ah sorry, I should have updated it in the readme to make it more clear.

Comment (magicksRampage) [reply ew.py:669]: was there a problem with storing coin gambled separately the way we have been?

Comment (EthanStraub) [reply ew.py:669]: Yes, Kanman (and Trackpad too I believe) already hit the cap on bigint.

Comment (EthanStraub) [reply ew.py:669]: To be more specific, Kanman can no longer gain festivity from gambling slimecoin, as he has reached the max possible value slimernalia_coin_gambled can store

**Pull request merged**